news/indir.rst

```diff
@@ -0,0 +1,14 @@
+**Added:**
+
+* ``indir`` a context manager for temporarily entering into a directory
+* ``run`` and ``check_call`` subprocess stubs using ``xonsh`` as the backend
```

Comment:
Can you give the full object path here, ie `xonsh.lib.indir`.  It would also probably be good to start with an item that says something like ```* New xonsh standard library `xonsh.lib` subpackage```